zubhub_frontend/zubhub/src/components/notification/Notification.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styles from '../../assets/js/styles/components/notification/NotificationStyles';
+import { makeStyles } from '@material-ui/core/styles';
+import { useSelector } from 'react-redux';
+import API from '../../api/api';
+import {
+  ListItem,
+  ListItemAvatar,
+  ListItemText,
+  Avatar,
+} from '@material-ui/core';
+import { AvatarGroup } from '@material-ui/lab';
+import { dFormatter } from '../../assets/js/utils/scripts';
+import { useTranslation } from 'react-i18next';
+import { Link, useHistory } from 'react-router-dom';
+
+const useStyles = makeStyles(styles);
+const Notification = ({ notification }) => {
+  const classes = useStyles();
+  const token = useSelector(store => store.auth.token);
+  const { t } = useTranslation();
+  const st = dFormatter(JSON.parse('"' + notification.date + '"'));
+
+  const getMessage = () => {
+    let message = '';
+    // 55 character message limit + 17 for strong tags being counted as well
+    if (notification.message.length < 72) {
+      message += notification.message;
+    } else {
+      message += notification.message.slice(0, 72) + '...';
+    }
+    return message;
+  };
+
```

Comment:
[This](https://stackoverflow.com/questions/15909489/text-overflow-ellipsis-on-two-lines) might be a nice option for text overflow.

Reply:
I looked at this a bit but it doesn't seem -webkit-box is compatible with JS styles